code/frameworks/vue3-vite/src/preset.ts

```diff
@@ -12,6 +12,7 @@ export const viteFinal: StorybookConfig['viteFinal'] = async (config, { presets
 
   // Add vue plugin if not present
   if (!(await hasVitePlugins(plugins, ['vite:vue']))) {
+    // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This bugged me, so I opened https://github.com/storybookjs/storybook/pull/20440.

Reply:
This ignore line will need to be removed now that the other PR is merged.